packages/store/ts/config/v2/input.ts

```diff
@@ -26,7 +26,7 @@ export type TablesInput = {
 
 export type StoreInput = {
   readonly namespace?: string;
-  readonly tables: TablesInput;
+  readonly tables: Omit<TablesInput, "name" | "namespace">;
```

Comment:
this is `TablesInput` so it's an object with tables, so how does this work? don't we have to map over it and omit? or put this in `TableInput`?

Reply:
also tiny nit: I tend to order these as namespace then name

Reply:
ah yes you're right, this doesn't do anything